setup_wordpress.sh

```diff
@@ -1,3 +1,245 @@
+CREATE USER 'wordpressuser'@'localhost' IDENTIFIED BY 'password';
+GRANT ALL PRIVILEGES ON wordpress.* to wordpressuser@'localhost';
+FLUSH PRIVILEGES;
+EOF
+
+echo "The Default Password Set For The 'WordPress' Database is 'password'. You Can Change It Later."
+
+echo "------------------ Installing PHP Extentions ------------------"
+sudo apt update
+sudo apt -y install php-curl php-gd php-intl php-mbstring php-soap php-xml php-xmlrpc php-zip
+sudo systemctl restart php7.4-fpm
+
+echo "------------------ Adding Location Blocks ------------------"
+echo "
+server {
+    listen 443 ssl;
+    listen [::]:443 ssl;
+    include snippets/self-signed.conf;
+    include snippets/ssl-params.conf;
+
+
+    server_name $ip_address;
+    root /var/www/wordpress;
+
+    index index.html index.htm index.php;
+
+    location / {
+        #try_files \$uri \$uri/ =404;
+         try_files \$uri \$uri/ /index.php\$is_args$args;
+    }
+
+    location ~ \.php$ {
+        include snippets/fastcgi-php.conf;
+        fastcgi_pass unix:/var/run/php/php7.4-fpm.sock;
+    }
+
+    location ~ /\.ht {
+        deny all;
+    }
+
+    location = /favicon.ico { log_not_found off; access_log off; }
+    location = /robots.txt { log_not_found off; access_log off; allow all; }
+    location ~* \.(css|gif|ico|jpeg|jpg|js|png)$ {
+        expires max;
+        log_not_found off;
+    }
+
+}
+
+server {
+    listen 80;
+    listen [::]:80;
+
+    server_name $ip_address;
+
+    return 302 https://\$server_name\$request_uri;
+
+}" | sudo tee /etc/nginx/sites-available/wordpress
+
+echo "------------------ Testing Nginx ------------------"
+sudo nginx -t
+
+echo "------------------ Restarting Nginx ------------------"
+sudo systemctl reload nginx
+
+echo "------------------ Downloading WordPress ------------------"
+cd /tmp
+curl -LO https://wordpress.org/latest.tar.gz
+
+echo "------------------ Extracting WordPress ------------------"
+tar xzvf latest.tar.gz
+sudo cp /tmp/wordpress/wp-config-sample.php /tmp/wordpress/wp-config.php
+sudo cp -a /tmp/wordpress/. /var/www/wordpress
+
+echo "------------------ Setting Up Privileges ------------------"
+sudo chown -R www-data:www-data /var/www/wordpress
+
+echo "------------------ Configuring WordPress ------------------"
+sudo cp /home/$USER/wp-config.php /var/www/wordpress/wp-config.php
+
+echo "------------------ Restarting Nginx ------------------"
+sudo systemctl restart nginx
+
+echo "------------------ Installing WordPress CLI ------------------"
+curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
+php wp-cli.phar --info
+sudo chmod +x wp-cli.phar
+sudo mv wp-cli.phar /usr/local/bin/wp
+wp cli version
+
+cd /var/www/wordpress
+echo "------------------ Creating WordPress User ------------------"
+wp core install --url="192.168.8.191" --title="WordPress Setup" --admin_user=admin --admin_email=admin@test.com --admin_password="!2three456." --allow-root
```

Comment:
previously you used variable `$ip_address` and here you hardcore the ip?

Reply:
So is it ok if I use the provided Vagrant IP without user interaction? Thanks for issuing that by the way.